Lib/test/test_cmd_line.py

```diff
@@ -955,7 +955,7 @@ def test_cmd_dedent(self):
             output2 = proc2.stdout.strip()
 
             self.assertEqual(output1, output2)
-            self.assertEqual(output1, case['expected'])
+            self.assertEqual(output1.replace(b'\r', b''), case['expected'])
```

Comment:
This behavior is inherited from `textwrap.dedent`, where only `\n' is considered.

Better to match `\r\n` instead of `\r` alone.

I can take a look at the dedent test cases in `test_textwrap.py`.

https://github.com/python/cpython/blob/2d526cd32fe8b286aae38956648e508070729f8f/Lib/test/test_textwrap.py#L766

Reply:
This was just normalizing the windows line endings into unix ones. I changed it to be more clear.